.github/workflows/test.yml

```diff
@@ -81,3 +81,27 @@ jobs:
         run: |
           source .venv/bin/activate
           tox -e ${{ matrix.toxenv }}
+      
+      - name: Coveralls Parallel
```

Comment:
If that's possible, Instead of `Parallel`, I would use something like `Coveralls for ${{ matrix.toxenv }}`

Reply:
oke, done